ravenframework/unSupervisedLearning.py

```diff
@@ -189,7 +189,7 @@ def train(self, tdict, metric=None):
 
     ## Check if a label feature is provided by the user and in the training data
     if self.labelFeature in tdict:
-      self.labelValues = tidct[self.labelFeature]
+      self.labelValues = tdict[self.labelFeature]
```

Comment:
How was this working? Did this if statement never get true in all the tests?

Reply:
it seems like the statement was not being hit in the tests, I stumbled upon that typo by chance